package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@microsoft/logicappsux",
-  "version": "2.84.0",
+  "version": "2.85.0",
```

Comment:
As this is my first package update in this repo, I wasn't sure if this change was supposed to happen, but it was made automatically when I did `npm install` after updating Lexical.